stdlib/public/Cxx/CxxSpan.swift

```diff
@@ -9,6 +9,13 @@
 // See https://swift.org/CONTRIBUTORS.txt for the list of Swift project authors
 //
 //===----------------------------------------------------------------------===//
+import Builtin
+
+public func unsafeBitCast<T: ~Escapable, U>(
+   _ x: T, to type: U.Type
+) -> U {
+  Builtin.reinterpretCast(x)
+}
```

Comment:
Can we use the existing `unsafeBitCast` without redeclaring it in this module?

Reply:
The problem is, currently `unsafeBitCast` is not updated to work with non-escapable types. Using it directly would result in this error:
```
global function 'unsafeBitCast(_:to:)' requires that 'Span<Self.Element>' conform to 'Escapable'
```

Hopefully, once https://github.com/swiftlang/swift/pull/73258 lands, we could remove this copy.

Reply:
Would this cause overload ambiguity when `unsafeBitCast` is used from client code? Could this overload be `internal`?

Reply:
Done.